pom.xml

```diff
@@ -444,6 +444,11 @@
 						</supportedProjectTypes>
 					</configuration>
 				</plugin>
+        <plugin>
```

Comment:
I'm not sure to see why we need this plugin.

Reply:
my bad, publishing locally ...